src/components/offline.js

```diff
@@ -0,0 +1,2 @@
+import { Offline } from 'react-detect-offline';
+export default Offline;
```

Comment:
I don't get it

Reply:
The dynamic import needs to load a component from a file. It's just a wrapper around the react dectect offline module